src/app/tl-drop-down/tl-drop-down.component.scss

```diff
@@ -1,30 +1,8 @@
 @import "../shared/styles/tl-variables.scss";
+@import "../shared/styles/tl-branded-btn.scss";
```

Comment:
Maybe we should include this in the host section.

Reply:
Adding this in the host is taking away the styling.